.gitignore

```diff
@@ -35,6 +35,7 @@ timeoverflow_development
 timeoverflow_test
 
 .env
+config/database.yml
```

Comment:
I use to have an untracked `database.yml` in my environment, so I can change it as my wish. 

On the other hand, people who can't copy a database.yml.sample to database.yml maybe should learn to do it..
Just uncomment these lines and :tada: https://github.com/coopdevs/timeoverflow/blob/master/bin/setup#L15-L18

I've been looking for the initial setup in the `Getting started guide` and seems we are not using the `bin/setup` script..  maybe it will make setup a little easier.